core/coreunix/add.go

```diff
@@ -120,6 +121,11 @@ func (adder *Adder) add(reader io.Reader) (ipld.Node, error) {
 		NoCopy:     adder.NoCopy,
 		CidBuilder: adder.CidBuilder,
 	}
+	if fi, ok := reader.(files.FileInfo); ok {
+		if absPath := fi.AbsPath(); filestore.IsURL(absPath) {
+			params.URL = absPath
+		}
+	}
```

Comment:
I'm not sure we need this given that we're setting AbsPath. We can probably get rid of this parameter all together and construct a `WebFile` in `core/commands/urlstore.go` instead.

Reply:
Would that cause the data to be downloaded again?

Reply:
In the `urlstore` command, we download the data on the server (I'm talking about two different commands, we can probably fix _this_ later).

Reply:
Ah I see - this code is completely redundant, and if we use a `WebFile` from `urlstore`, then the URL param is unused. Would `urlstore` itself be redundant also though? Is it still worth keeping around?

Reply:
urlstore will be redundant but we should keep it around for compatibility (for a few releases as it _is_ experimental).